overlord/snapshotstate/snapshotstate.go

```diff
@@ -510,6 +510,13 @@ func Restore(st *state.State, setID uint64, snapNames []string, users []string)
 		ts.AddTask(task)
 	}
 
+	if len(summaries) > 0 {
+		desc := fmt.Sprintf("Cleanup of restore snapshot set #%d", setID)
```

Comment:
this merits a comment:

// take care of cleaning up all restore working state if all the restore tasks succeeded,
// if they didn't the undo logic will take care of this